### PR TITLE
Add QT_FORCE_OPENGL_TYPE

### PR DESCRIPTION
**Pull request by Dami-star**
The user can set QT_FORCE_OPENGL_TYPE to OpenGL or OpenGLES. The default value is OpenGL. No longer need to reconfigure the compilation parameter es2 opengl.

### CODE REVIEW

File: debian/patches/add-QT_RENDER_TYPE.patch [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ #include <QtGui/qpixmapcache.h>
++#include <QtGui/QSurfaceFormat>
+ #include <qpa/qplatforminputcontext.h>
+ #include <qpa/qplatforminputcontext_p.h>
+ 
+@@ -1435,6 +1436,17 @@ void QGuiApplicationPrivate::createPlatf
+     // Load the platform integration
+     QString platformPluginPath = QString::fromLocal8Bit(qgetenv("QT_QPA_PLATFORM_PLUGIN_PATH"));
+ 
++    QString renderType = QString::fromLocal8Bit(qgetenv("QT_RENDER_TYPE"));
```
Comment (zccrs) [inline debian/patches/add-QT_RENDER_TYPE.patch:22]: Use `QT_FORCE_OPENGL_TYPE`

Comment (Dami-star) [reply debian/patches/add-QT_RENDER_TYPE.patch]: done

File: debian/patches/series [hunk 2]
```diff
@@ -22,3 +22,4 @@ Make-sure-hicolor-searched-before-dashfallbacks.patch
 
 # https://codereview.qt-project.org/c/qt/qtbase/+/526124
 add_QT_IM_MODULES_env_to_allows_multi_IM_key.patch
+add-QT_RENDER_TYPE.patch
```
Comment (zccrs) [inline debian/patches/series:25]: patch file name

**Review by zccrs — CHANGES_REQUESTED** [review]

File: debian/patches/series (hunk 2 above)
Comment (Dami-star) [reply debian/patches/series]: done

**Review by zccrs — APPROVED** [review]

**Pull request merged**